ui/apos/components/AposImportModal.vue

```diff
@@ -177,13 +189,23 @@ export default {
         return;
       }
       apos.bus.$emit('import-export-import-started');
+
+      const notifId = await apos.notify('apostrophe:uploading', {
+        type: 'progress',
+        icon: 'cloud-upload-icon',
+        interpolate: {
+          name: this.selectedFile.name
+        }
+      });
+
       bigUpload(`${this.universalModuleAction}/${this.action}`, {
         files: {
           file: this.selectedFile
         },
         body: {
           importDraftsOnly: this.checked.includes('importDraftsOnly')
-        }
+        },
+        progress: this.instantiateProcess(notifId)
```

Comment:
Semicolon

Reply:
It's a property of the object passed to `bigUpload`.